### ISSUE TITLE
20200505 Github 前端趋势榜

### ISSUE THREAD

**Issue by kujian** [kujian/githubTrending#269]
# 20200505 Github 前端趋势榜
[Github 前端开发日报](http://caibaojian.com/c/news)栏目数据来自[Github热榜](http://news.caibaojian.com/)（我开发的爬虫），每日分享HTML、CSS、JavaScript、Vue等开源项目，为开发者提供动力，点击Star按钮来关注这个项目，点击Watch来收听每日的更新[Github主页](https://github.com/kujian/githubTrending)
## JavaScript
* [poloclub/cnn-explainer](https://github.com/poloclub/cnn-explainer) （Learning Convolutional Neural Networks with Interactive Visualization. https://poloclub.github.io/cnn-explainer/）
* [yangshun/tech-interview-handbook](https://github.com/yangshun/tech-interview-handbook) （? Algorithms, front end and behavioral content for rocking your coding interview）
* [alpinejs/alpine](https://github.com/alpinejs/alpine) （A rugged, minimal framework for composing JavaScript behavior in your markup.）
* [facebook/create-react-app](https://github.com/facebook/create-react-app) （Create React apps with no build configuration.）
* [lyswhut/lx-music-desktop](https://github.com/lyswhut/lx-music-desktop) （一个基于 electron 的音乐下载软件）
* [axios/axios](https://github.com/axios/axios) （Promise based HTTP client for the browser and node.js）
* [gothinkster/realworld](https://github.com/gothinkster/realworld) （        A Collection &amp; Specification for Exemplary Frontend and Backend Codebases ?      ）
* [laurent22/joplin](https://github.com/laurent22/joplin) （Joplin - a note taking and to-do application with synchronization capabilities for Windows, macOS, Linux, Android and iOS.）
* [facebook/react-native](https://github.com/facebook/react) （        A framework for building native apps with React.      ）
* [goldbergyoni/nodebestpractices](https://github.com/goldbergyoni/nodebestpractices) （✅ The largest Node.js best practices list (July 2019)）
* [cyrildiagne/ar-facedoodle](https://github.com/cyrildiagne/ar-facedoodle) （Doodle AR masks right from the browser with tensorflow.js）
* [mikebryant/ac-nh-turnip-prices](https://github.com/mikebryant/ac-nh-turnip-prices) （Price calculator/predictor for Turnip prices）
* [iamadamdev/bypass-paywalls-chrome](https://github.com/iamadamdev/bypass-paywalls-chrome) （Bypass Paywalls for Chrome）
* [cypress-io/cypress](https://github.com/cypress-io/cypress) （Fast, easy and reliable testing for anything that runs in a browser.）
* [Fugiman/google-meet-grid-view](https://github.com/Fugiman/google-meet-grid-view) （Userscript to offer a grid-view layout in Google Meets）
* [serverless/serverless](https://github.com/serverless/serverless) （        Serverless Framework – Build web, mobile and IoT applications with serverless architectures using AWS Lambda, Azure Functions, Google CloudFunctions &amp; more! –       ）
* [TheAlgorithms/Javascript](https://github.com/TheAlgorithms/Javascript) （A repository for All algorithms implemented in Javascript (for educational purposes only)）
* [plotly/plotly.js](https://github.com/plotly/plotly.js) （Open-source JavaScript charting library behind Plotly and Dash）
* [sequelize/sequelize](https://github.com/sequelize/sequelize) （An easy-to-use multi SQL dialect ORM for Node.js）
* [jhu-ep-coursera/fullstack-course4](https://github.com/jhu-ep-coursera/fullstack-course4) （Example code for HTML, CSS, and Javascript for Web Developers Coursera Course）
* [wix/Detox](https://github.com/wix/Detox) （Gray box end-to-end testing and automation framework for mobile apps）
* [MrRio/jsPDF](https://github.com/MrRio/jsPDF) （Client-side JavaScript PDF generation for everyone.）
* [covid19india/api](https://github.com/covid19india/api) （）
* [open-coronavirus/open-coronavirus](https://github.com/open-coronavirus/open-coronavirus) （Open Coronavirus）
* [angular/angular.js](https://github.com/angular/angular) （        AngularJS - HTML enhanced for web apps!      ）

## Vue
* [ChiChou/Grapefruit](https://github.com/ChiChou/Grapefruit) （(WIP) Next generation Passionfruit. Runtime Application Instruments for iOS）
* [liyasthomas/postwoman](https://github.com/liyasthomas/postwoman) （&#x1f47d; API request builder - Helps you create your requests faster, saving you precious time on your development.）
* [mtolhuys/laravel-schematics](https://github.com/mtolhuys/laravel-schematics) （A Laravel package making a diagram of your models, relations and the ability to build them with it）
* [shidoitsuka/hi-ava](https://github.com/shidoitsuka/hi-ava) （I made this for her. You can make yours for yours, too!）
* [liuzhuoling2011/baidupcs-web](https://github.com/liuzhuoling2011/baidupcs-web) （基于vue2+nuxt构建的高仿饿了么（2018版））
* [GitHubGanKai/vue3-jd-h5](https://github.com/GitHubGanKai/vue3-jd-h5) （基于vue2.6+ ，vue3.0.0-beta.1，vue3, vue-cli3，mockjs，仿京东淘宝的，移动端H5电商平台！）
* [Requarks/wiki](https://github.com/Requarks/wiki) （Wiki.js | A modern, lightweight and powerful wiki app built on NodeJS, Git and Markdown）
* [mandatoryprogrammer/ChromeGalvanizer](https://github.com/mandatoryprogrammer/ChromeGalvanizer) （Harden your Chrome browser via enterprise policy.）
* [lusaxweb/vuesax](https://github.com/lusaxweb/vuesax) （Frontend vue Components）
* [buefy/buefy](https://github.com/buefy/buefy) （Lightweight UI components for Vue.js based on Bulma）
* [YaoZeyuan/stablog](https://github.com/YaoZeyuan/stablog) （稳部落. 专业备份导出微博记录, 稳!）
* [codigofalado/obs-countdown](https://github.com/codigofalado/obs-countdown) （Tela de Countdown para Streams）
* [Armour/vue-typescript-admin-template](https://github.com/Armour/vue-typescript-admin-template) （&#x1f596; A vue + typescript minimal admin template）
* [coderwhy/HYMall](https://github.com/coderwhy/HYMall) （A vue.js mall application.）
* [fox-one/mornin.fm](https://github.com/fox-one/mornin.fm) （）
* [coreui/coreui-free-vue-admin-template](https://github.com/coreui/coreui-free-vue-admin-template) （CoreUI Vue is free Vue admin template based on Bootstrap 4）
* [jae-jae/Userscript-Plus](https://github.com/jae-jae/Userscript-Plus) （&#x1f412; Show current site all UserJS，The easier way to install UserJs for Tampermonkey. 显示当前网站的所有可用Tampermonkey脚本）
* [Microsoft/AzureTipsAndTricks](https://github.com/Microsoft/AzureTipsAndTricks) （Learn some of our favorite Azure tips and tricks—some long-standing, and new ones that have recently been added to become more productive with Azure. Star the repo now to save hours off your coding tasks tomorrow.）
* [ghosh/uiGradients](https://github.com/ghosh/uiGradients) （        Beautiful colour gradients for design and code      ）
* [jsbroks/coco-annotator](https://github.com/jsbroks/coco-annotator) （✏️ Web-based image segmentation tool for object detection and localization）
* [tuandm/laravue](https://github.com/tuandm/laravue) （Laravel dashboard built by VueJS - http://laravue.cipherpols.com）
* [fenixsoft/awesome-fenix](https://github.com/fenixsoft/awesome-fenix) （讨论与实践单体、微服务、无服务等架构风格的开源文档：《软件架构探索：The Fenix Project》）

## HTML
* [docker/awesome-compose](https://github.com/docker/awesome-compose) （Awesome Docker Compose samples）
* [nndl/nndl.github.io](https://github.com/nndl/nndl.github.io) （《神经网络与深度学习》 Neural Network and Deep Learning）
* [h5bp/Front-end-Developer-Interview-Questions](https://github.com/h5bp/Front-end-Developer-Interview-Questions) （        A list of helpful front-end related questions you can use to interview potential candidates, test yourself or completely ignore.      ）
* [ncase/covid-19](https://github.com/ncase/covid-19) （COVID-19 Futures, Explained With Playable Simulations）
* [RayTracing/raytracing.github.io](https://github.com/RayTracing/raytracing.github.io) （Main Repo / Main Web Site）
* [LAB-MI/deplacement-covid-19](https://github.com/LAB-MI/deplacement-covid-19) （Service de génération de l'attestation de déplacement dérogatoire à présenter dans le cadre du confinement lié au virus covid-19）
* [freddier/hyperblog](https://github.com/freddier/hyperblog) （Un blog increíble para el curso de Git y Github de Platzi）
* [rfordatascience/tidytuesday](https://github.com/rfordatascience/tidytuesday) （Official repo for the #tidytuesday project）
* [OWASP/owasp-mstg](https://github.com/OWASP/owasp-mstg) （         The Mobile Security Testing Guide (MSTG) is the ultimate guide for mobile app security testing and reverse engineering.      ）
* [acgeospatial/awesome-earthobservation-code](https://github.com/acgeospatial/awesome-earthobservation-code) （curated list of awesome tools, tutorials, code, helpful projects, links, stuff about Earth Observation and Geospatial stuff!）
* [DarkSecDevelopers/HiddenEye](https://github.com/DarkSecDevelopers/HiddenEye) （Modern phishing tool with advanced functionality）
* [svartmc/trails](https://github.com/svartmc/trails) （Simple geometrical trail to attach to your Three.js objects）
* [google/fonts](https://github.com/google/fonts) （Font files available from Google Fonts）
* [wx-chevalier/Awesome-CS-Books](https://github.com/wx-chevalier/Awesome-CS-Books) （&#x1f4da; Awesome CS Books/Series(.pdf by git lfs) Warehouse for Geeks, ProgrammingLanguage, SoftwareEngineering, Web, AI, ServerSideApplication, Infrastructure, FE etc. &#x1f4ab; 优秀计算机科学与技术领域相关的书籍归档。）
* [phpstan/phpstan](https://github.com/phpstan/phpstan) （        PHP Static Analysis Tool - discover bugs in your code without running it!      ）
* [firebase/quickstart-js](https://github.com/firebase/quickstart-js) （        Firebase Quickstart Samples for Web      ）
* [alex-shpak/hugo-book](https://github.com/alex-shpak/hugo-book) （Hugo documentation theme as simple as plain book）
* [londonappbrewery/TinDog-Start](https://github.com/londonappbrewery/TinDog-Start) （）
* [nax3t/webdevbootcamp](https://github.com/nax3t/webdevbootcamp) （All source code for back-end projects from the Web Developer Bootcamp）
* [csev/py4e](https://github.com/csev/py4e) （Web site for www.py4e.com and source to the Python 3.0 textbook）

## CSS
* [Ajusa/lit](https://github.com/Ajusa/lit) （? CSS framework）
* [astrit/css.gg](https://github.com/astrit/css.gg) （500+ CSS Icons. Customizable, Retina Ready with API &amp; NPM）
* [sliminality/ply](https://github.com/sliminality/ply) （CSS inspection aided by visual regression pruning）
* [daneden/animate.css](https://github.com/daneden/animate.css) （        A cross-browser library of CSS animations. As easy to use as an easy thing.      ）
* [necolas/normalize.css](https://github.com/necolas/normalize.css) （A collection of HTML element and attribute style-normalizations      ）
* [dortania/OpenCore-Desktop-Guide](https://github.com/dortania/OpenCore-Desktop-Guide) （Repo for the OpenCore Desktop Guide）
* [easychen/howto-make-more-money](https://github.com/easychen/howto-make-more-money) （程序员如何优雅的挣零花钱）
* [zhangyingwei/html-css-only](https://github.com/zhangyingwei/html-css-only) （漂亮的 CSS 系列~）
* [bastienwirtz/homer](https://github.com/bastienwirtz/homer) （A very simple static homepage for your server.）
* [pmarsceill/just-the-docs](https://github.com/pmarsceill/just-the-docs) （A nice looking, high customizable, responsive Jekyll theme for documention with built-in search.）
* [robb0wen/synthwave-vscode](https://github.com/robb0wen/synthwave-vscode) （Synthwave inspired colour theme for VS Code &#x1f305;&#x1f576;）
* [theme-next/hexo-theme-next](https://github.com/theme-next/hexo-theme-next) （Elegant and powerful theme for Hexo.）
* [KyleAMathews/typefaces](https://github.com/KyleAMathews/typefaces) （NPM packages for Open Source typefaces）
* [Microsoft/TypeScript-Node-Starter](https://github.com/Microsoft/TypeScript-Node-Starter) （        A starter template for TypeScript and Node with a detailed README describing how to use the two together.      ）
* [RyanFitzgerald/devportfolio](https://github.com/RyanFitzgerald/devportfolio) （A lightweight, customizable single-page personal portfolio website template built with JavaScript and Sass）
* [juliocesarfort/public-pentesting-reports](https://github.com/juliocesarfort/public-pentesting-reports) （Curated list of public penetration test reports released by several consulting firms and academic security groups）
* [thewhiteh4t/seeker](https://github.com/thewhiteh4t/seeker) （Accurately Locate Smartphones using Social Engineering）
* [json-api/json-api](https://github.com/json-api/json-api) （        A specification for building JSON APIs      ）

## TypeScript
* [cyrildiagne/ar-cutpaste](https://github.com/cyrildiagne/ar-cutpaste) （Cut and paste your surroundings using AR）
* [phuoc-ng/1loc](https://github.com/phuoc-ng/1loc) （What's your favorite JavaScript single LOC (line of code)?）

## Unknow
* [sudheerj/reactjs-interview-questions](https://github.com/sudheerj/reactjs-interview-questions) （List of top 500 ReactJS Interview Questions &amp; Answers....Coding exercise questions are coming soon!!）
* [kamranahmedse/developer-roadmap](https://github.com/kamranahmedse/developer-roadmap) （        Roadmap to becoming a web developer in 2017      ）
* [florinpop17/app-ideas](https://github.com/florinpop17/app-ideas) （A Collection of application ideas which can be used to improve your coding skills.）
* [CSSEGISandData/COVID-19](https://github.com/CSSEGISandData/COVID-19) （Novel Coronavirus (COVID-19) Cases, provided by JHU CSSE）
* [ossu/computer-science](https://github.com/ossu/computer-science) （? Path to a free self-taught education in Computer Science!）
* [getify/You-Dont-Know-JS](https://github.com/getify/You-Dont-Know-JS) （        A book series on JavaScript. @YDKJS on twitter.      ）

## 关于我
 [前端开发博客](http://caibaojian.com/) 站长，[微博](https://weibo.com/kujian) 粉丝18万，[微信公众号](http://t.cn/EV7pcM1)。
如果你觉得我的分享对你有帮助，不妨考虑打赏我[一杯咖啡](http://caibaojian.com/donation)，以资鼓励。
![打赏](https://upload-images.jianshu.io/upload_images/570843-db4053c67a8c9ea9.png)
## 推荐
1. [掘金小册8折优惠，还有全网优惠教程](http://caibaojian.com/goods)
2. [加我微信，进500人微信群，领100本前端电子书](http://t.cn/EV7pcM1)
## 微信群
加入微信群，每天晚上我会汇总一天最好的文章给你。
![qianduanquan](https://user-images.githubusercontent.com/3055447/38468989-651132ac-3b80-11e8-8e6b-15122322a9d7.png)
